nifi-assembly/pom.xml

```diff
@@ -862,6 +862,12 @@ language governing permissions and limitations under the License. -->
             <version>1.18.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-jslt-nar</artifactId>
+            <version>1.16.0-SNAPSHOT</version>
```

Comment:
Looks like this needs to be updated to `1.18.0-SNAPSHOT`.

Reply:
Will update to 1.19.0-SNAPSHOT